core-addon/IonCore.js

```diff
@@ -12,7 +12,7 @@ const ION_DEFAULT_ARGS = {
   // NOTE: if this URL ever changes, you will have to update the domain in
```

Comment:
Do you want to rename `IonCore.js` as well or not yet?

Reply:
Also `./tests/core-addon/IonCore.test.js`

Reply:
I'll be dealing with other code changes in #114 if you don't mind, as this PR already got really big :(